packages/documentation/docs/controls/_pane_styleguide.md

```diff
@@ -0,0 +1,55 @@
+Panes are interactive component that enable users to access hidden content or options that are not immediately visible on the screen. These drawers are usually hidden behind a button or icon and can be revealed by clicking on the control element. Panes can be used for organizing complex information, displaying multiple views or sections, collapsible or expandable, but not the solution for limited content, or confusing layout.
```

Comment:
- component**s**
- Consistent wording: on Blind chapter it says "allow" instead of "enable"
- I would avoid using the term "drawer" here since this is a separate (though deprecated) component
- "confusing layout" => this is a general UX thing and not really connected to the panes

What I'm missing on the page is the actual awesome use cases and advantages, like:
- better task fokus -> related controls can be visually grouped, the main content is more tidied up
- more efficient use of screen space -> more compact and organized manner, especially for smaller screens
- more dynamic layouting option and visualizing hierarchy - than just everything in the main content
- gives contextual information -> often, panes contain contextual details e.g. about a selected option in the UI